RELEASE_NOTES.md

```diff
@@ -1,5 +1,8 @@
 # Release Notes
 
+## 6.0.1 - 2024-07-11
+* ENHANCEMENT: Fixes for usage in .NET 8.0 enviroment projects.
+
 ## 6.0.1 - 2024-06-04
 * BUGFIX: MSBuild.build adds a bad string at the end of properties, thanks @0x53A - https://github.com/fsprojects/FAKE/issues/2738
 * ENHANCEMENT: Added shorthash to git functions, thanks @voronoipotato - https://github.com/fsprojects/FAKE/pull/2752
```

Comment:
I think it must go together as one. @Numpsy right?
```suggestion
## 6.0.1 - 2024-07-11
* BUGFIX: MSBuild.build adds a bad string at the end of properties, thanks @0x53A - https://github.com/fsprojects/FAKE/issues/2738
* ENHANCEMENT: Added shorthash to git functions, thanks @voronoipotato - https://github.com/fsprojects/FAKE/pull/2752
* ENHANCEMENT: Fixes for usage in .NET 8.0 enviroment projects.
```

Reply:
I think so